response_operations_ui/controllers/admin_controller.py

```diff
@@ -41,3 +45,25 @@ def current_banner():
         return banner
     except redis.RedisError:
         logger.exception("Unable to retrieve current banners")
+
+
+def banner_time_get():
+    try:
+        r = _get_redis()
+        banner = r.get('AVAILABILITY_MESSAGE_TIME_SET')
+        logger.debug("Getting time availability message was set", banner=banner)
+        return banner
+    except redis.RedisError:
+        logger.exception("Unable to retrieve current banners")
+
+
+def get_alert_list():
+    my_dict = {}
```

Comment:
As Warren suggested before, remove the my_dict stuff

Reply:
This change investigated. A list of dicts is returned instead of a dictionary, this required extra code in banner-admin.html. It was agreed to leave it like this for readability within the html file - happy to make the change though

Reply:
👍